### PR TITLE
add delete contact item logic

### PR DESCRIPTION
**Pull request by Yana-Kalandarova**

### CODE REVIEW

File: src/components/DeleteBtn.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import deleteIcon from '../assets/delete-icon.svg';
+
+const propTypes = {
+  onDeleteItem: PropTypes.func.isRequired,
+  id: PropTypes.number.isRequired,
+};
+
+function DeleteBtn({ onDeleteItem, id }) {
```
Comment (maximshenets) [inline src/components/DeleteBtn.js]: Better to rename it to `DeleteButton`

File: src/components/app.js [hunk 2]
```diff
@@ -3,13 +3,35 @@ import '../styles/app.scss';
...
+
+    this.state = {
+      contactList: dataList,
+    };
+  }
+
+  handleDeleteItem = (id) => {
+    const { contactList } = this.state;
+
+    const deletedIndex = contactList.findIndex(el => el.id === id);
```
Comment (maximshenets) [inline src/components/app.js]: Confused name `el`, can be `contact`

File: src/components/app.js [hunk 3]
```diff
@@ -3,13 +3,35 @@ import '../styles/app.scss';
...
 import ContactList from './ContactList';
 
-function App() {
-  return (
-    <div className="view_wrap">
-      <h1 className="view_title">Contact List</h1>
-      <ContactList list={dataList} />
-    </div>
-  );
+class App extends React.Component {
```
Comment (maximshenets) [inline src/components/app.js]: You can use destructuring in `import React, { Component } from 'react'`

File: src/components/DeleteBtn.js (hunk 1 above)
Comment (Yana-Kalandarova) [reply src/components/DeleteBtn.js]: Renamed.

File: src/components/app.js (hunk 2 above)
Comment (Yana-Kalandarova) [reply src/components/app.js]: Renamed.

File: src/components/app.js (hunk 3 above)
Comment (Yana-Kalandarova) [reply src/components/app.js]: Done.

**Pull request merged**